packages/storage/src/bucket.js

```diff
@@ -1428,6 +1428,46 @@ Bucket.prototype.setMetadata = function(metadata, options, callback) {
 };
 
 /**
+ * Set the storage class for this bucket.
```

Comment:
Set a buckets default storage class for new files.  https://cloud.google.com/storage/docs/json_api/v1/buckets/update#storageClass